src/bahmni.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# Description taken throuhg a synthesis of the github 
+# description and their FAQ
+description: >
+  Bahmi is an Open Source hospital Management System focusing
+  on poor/underserved and public hospitals in the developing
+  world.
+  It's aimed to being a generic system which can be used for
+  multiple diseases and hospitals in different countries.
+license: AGPL-3.0
+name: Bahmni
+programmingLanguages:
+  - java
+  - javascript
+  - python
+  - shell
+  - groovy
```

Comment:
Note that these are only the Top languages listed in github. I can see other projects with TSQL, Ruby, Perl and SQLPL.
That's also why I thought of adding the "main: true" optional argument. Some projects are mainly 1 language (eg Java), with a bunch of other languages for minor scripting.

Reply:
If you want we can re-introduce `main`. I was only against having only one language with it, not multiple.
Would you mind adding an exact definition somewhere of what warrants a `main` tag to you? Just so we can stay consistent :)